styleguide/derek/examples/stories-interior.ejs

```diff
@@ -0,0 +1,49 @@
+<%- partial("../_partials/_ceiling") %>
+<%- partial("../_partials/_mainnav") %>
+<%- partial('../_partials/solutions/header-smallIMAC', { headline: 'Customer Stories'}) %>
```

Comment:
http://localhost:9000/derek/examples/imac
http://localhost:9000/derek/assets/photography

Those pages have small banners that were broken by this update to the partial.